src/backend/distributed/shared_library_init.c

```diff
@@ -2722,4 +2724,17 @@ CitusObjectAccessHook(ObjectAccessType access, Oid classId, Oid objectId, int su
 		 * regardless if it's citus being created */
 		SetCreateCitusTransactionLevel(GetCurrentTransactionNestLevel());
 	}
+	else if (access == OAT_DROP && classId == ConstraintRelationId &&
```

Comment:
what about OAT_CREATE ?

Reply:
oops, you already noted that this doesn't care about OAT_CREATE yet

Reply:
oops, you already mentioned that this currently doesn't care about OAT_CREATE in PR description

Reply:
oops, you already mentioned that this currently doesn't care about OAT_CREATE in PR description